Rentit-frontend/src/components/items/items.jsx

```diff
@@ -1,23 +1,43 @@
 import './items.css';
 import { useNavigate } from 'react-router-dom';
-import fakedata from '../../fakedata.json';
+import { useEffect, useState } from 'react';
+
 const Items = () => {
+  const [itemData, setItemData] = useState(null);
+
+  useEffect(() => {
+    const url = 'http://127.0.0.1:2000/api-rentit/v1/items';
```

Comment:
domain url, api keys and other sensitive details should be stored in .evn files and access using process.env